feature-utils/poly-look/src/react-components/overlays/sideSheet.css

```diff
@@ -0,0 +1,34 @@
+.poly-side-sheet {
+  max-width: 288px;
```

Comment:
Is there any reason for this? It's not going to look good in an iPad, or even a phone in landscape mode.

Reply:
Right now all of our CSS targets lower aspect ratios, can't find where we documented this... But I'd say it's OK for now. If we want to support larger screens, we should find that documentation, update it, and then start testing for it, too.